file/src/test/scala/akka/stream/alpakka/file/impl/archive/ZipArchiveFlowTest.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.file.impl.archive
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Keep
+import akka.stream.testkit.scaladsl.{TestSink, TestSource}
+import akka.testkit.TestKit
+import akka.util.ByteString
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{Matchers, WordSpecLike}
+
+class ZipArchiveFlowTest extends TestKit(ActorSystem("ziparchive")) with WordSpecLike with Matchers with ScalaFutures {
```

Comment:
Please add a shutdown of the actor system.

```scala
  override def afterAll(): Unit = {
    super.afterAll()
    TestKit.shutdownActorSystem(system)
  }
```

Reply:
Fixed. In `ArchiveSpec` there was this issue too, so I fixed it too.